src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1446,6 +1501,7 @@ class ContentExplorer extends Component<Props, State> {
                             tableRef={this.tableRef}
                             view={view}
                             viewMode={viewMode}
+                            metadataColumnsToShow={metadataColumnsToShow}
```

Comment:
Is this going to work with #1598?

Reply:
It will if we don't go the inline editing route.